packages/dds/tree/src/core/rebase/utils.ts

```diff
@@ -307,7 +351,17 @@ export function rebaseChange<TChange>(
 		inverseFromCommit(changeRebaser, commit, mintRevisionTag, true),
 	);
 	inverses.reverse();
-	return rebaseChangeOverChanges(changeRebaser, change, [...inverses, ...targetPath]);
+
+	const telemetryProperties = {
+		sourceBranchLength: sourcePath.length,
+		rebaseDistance: targetPath.length,
```

Comment:
While this value describes the length of the branch that the rebased commit is from, we don't actually rebase that branch. Instead, we rebase a single change over `[...inverses, ...targetPath]`.

So I think what we should return is:
```suggestion
		sourceBranchLength: 1,
		rebaseDistance: sourcePath.length + targetPath.length,
```

Reply:
Thanks for catching this 👍 !